apps/core-app/src/main/modules/tray/tray-manager.test.ts

```
@@ -135,6 +137,37 @@ vi.mock('./tray-icon-provider', () => ({
 
 import { TrayManager } from './tray-manager'
 
+type TrayManagerOnInitHarness = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(ts/consistent-type-definitions)